cadence/templates/server-configmap.yaml

```diff
@@ -8,83 +8,98 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
 data:
-  docker_template.yaml: |-
+  docker_template_cassandra.yaml: |-
```

Comment:
Init script on cadence docker image expects config name format to be as following `docker_template_$DB.yaml`. Reference: https://github.com/uber/cadence/blob/master/docker/start.sh#L25

Reply:
Looks like 0.5.7 was released, so with a new minor chart version this should be okay.

Reply:
Still some fixes are not present in 0.5.7. Prometheus metrics are working only with master for now.